### PR TITLE
DGJ-1427 | add option param to hide the 3 dots loader in customEvent

### PR DESCRIPTION
**Pull request by jerry-ey**
## Summary
image

Removing 3 dots loader and only keep the "Submitting loader".

### CODE REVIEW

File: forms-flow-web/src/components/ServiceFlow/details/ServiceTaskDetails.js [hunk 1]
```diff
@@ -242,7 +242,8 @@ const ServiceFlowTaskDetails = React.memo(() => {
 
   const onFormSubmitCallback = (actionType = "", successPage) => {
     if (bpmTaskId) {
-      dispatch(setBPMTaskDetailLoader(true));
+      // comment out the following line. It is showing the 3 dots animation.
+      // dispatch(setBPMTaskDetailLoader(true));
```
Comment (iman-jamali-fw) [inline forms-flow-web/src/components/ServiceFlow/details/ServiceTaskDetails.js:246]: @jerry-ey 
If we disable the default loaded, does it mean all the forms should send an event to trigger the loader onSubmission?

Can you please test other forms with this change to see their current behaviour doesn't break? I belive only Telework form send event for loader.

Comment (jerry-ey) [reply forms-flow-web/src/components/ServiceFlow/details/ServiceTaskDetails.js:246]: Good call, I will double check and make necessary changes.

**Comment by jerry-ey** [comment]
Some forms don't have customEvent, so in order not breaking other form, this change will only apply to Mat/Pat with new param added to customEvent.

Will add this `isDefaultLoaderHidden: true` to DEV and TEST. Please review again.

image

**Review by iman-jamali-fw — APPROVED** [review]
Looks great Jerry!
Please just document the process before merging. Thanks

**Comment by jerry-ey** [comment]
> Looks great Jerry! Please just document the process before merging. Thanks

No problem. Will add the detail.

**Comment by jerry-ey** [comment]
Added document.

**Review by bhumin-fw — APPROVED** [review]
Looks good!

**Pull request merged**